examples/package.yaml

```diff
@@ -0,0 +1,3 @@
+dependencies:
+  btest:
```

Comment:
Basically says, that there is a package called `btest` at location `..`.

I did it as follows:
```
toit.pkg init # In the examples folder.
toit.pkg install --local --name=btest ..
```